tests/src/Opulence/Environments/EnvironmentTest.php

```diff
@@ -37,13 +37,7 @@ public function testGettingVariable()
      */
     public function testIsRunningInConsole()
     {
-        $isRunningInConsole = Environment::isRunningInConsole();
-
-        if (php_sapi_name() == "cli") {
-            $this->assertTrue($isRunningInConsole);
-        } else {
-            $this->assertFalse($isRunningInConsole);
-        }
+        $this->assertEquals(Environment::isRunningInConsole(), php_sapi_name() === "cli");
```

Comment:
Why do you say it's useless?  Yes, you're usually testing this in a console, so it's only checking that it evaluates to `true`, but isn't that better than not checking it all?